releasenotes/notes/add-run-async-tool-invoker-c3595e94f95636e4.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Added `run_async` method to `ToolInvoker` class to allow asynchronous tool invocations.
```

Comment:
Could we also add a note that Agent when running async will now also be able to stream the tool result?